### ISSUE TITLE
Turn this into errroverse meta-package

### ISSUE THREAD

**Issue by LukasWallrich** [ianhussey/ERROR#2]
... as umbrella over

- *Core*:
    - statcheck
    - scrutiny
    -  recalc

- *Auxiliary*:
    - rsprite2
    - within
    - faux
    - 

Questions:
- Include packages to deal with raw data (e.g., jfa), or just summary stats?
- Include package for PDF conversion? Or just best-practice guidance?

**Comment by lhdjung** [comment]
We should be careful before including packages that we don't maintain ourselves. We don't have any control over them, so if they are meant to interlock as in the tidyverse, we will only be able to architect that for our own packages. Also, we should check if a metapackage over non-self-maintained packages is possible on CRAN.

"Auxiliary" implies helper status. I think rsprite2 is more than that 😀 It seems like an important high-level, user-facing package, so why not core?

If we create lower-level backend packages (#6), we would need a third category.

**Comment by lhdjung** [comment]
If we don't bundle the packages of other people, they can still be great targets of documentation for us!

**Comment by ianhussey** [comment]
I agree that it would be great to get Michele on board, but yes we can't unilaterally include statcheck.

> Include packages to deal with raw data (e.g., jfa), or just summary stats?

Let's start smaller, within the inspection of articles without assuming access to the data. 

> Include package for PDF conversion? Or just best-practice guidance?

I'm imagining that users will extract data from articles manually. To keep scope narrower, at least to start, I would avoid trying to do PDF extractions within the package. I would rather see functions applied to manually extracted data, followed by a Shiny app using them, before trying to do anything on pdfs.

**Comment by LukasWallrich** [comment]
Thanks to both of you. 

We could certainly include other packages into an umbrella, though we obviously can't align their interface with ours (just like many packages reexport functions from others, or even just load other packages on load) - I don't think "meta-packages" are a distinct category ... but I get it that some people might see this framing as an appropriation of their work, so we should not do it without their explicit consent. If they (for now mostly Michele) are happy with it, but don't want to rework their packages, then we can discuss whether we still want to present them as part of the `errorverse`, or as recommended additional packages - not sure if that makes much of a difference. @ianhussey do you want to reach out to Michele to see whether she'd be interested in a collaboration on this? (She might still be on maternity leave though, I think I saw sth on Twitter)

I agree to focus on the articles, and on extracted data for now - getting that set up cleanly is enough work :)
